res/openvpn-gui-res-ru.rc

```diff
@@ -346,8 +348,8 @@ BEGIN
                   "--preconnectscript_timeout\t: Время ожидания завершения скрипта перед подключением.\n"
 
     IDS_NFO_USAGECAPTION "Использование OpenVPN GUI"
-    IDS_ERR_BAD_PARAMETER "Попытка обработать ""%s"" как --опция параметр, " \
-                      "но нет начальных '--'"
+    IDS_ERR_BAD_PARAMETER "Не удалось обработать строку ""%s"", которая была интерпретирована как опция вида ""--опция параметр"". " \
```

Comment:
Previous translation was too direct/straightforward. Person who don't know how getopt works wouldn't understand that.
@chipitsine, are you ok with such translation?